### DISCUSSION TITLE
Translation support?

### DISCUSSION THREAD

**Discussion by noum** [sa-si-dev/virtual-select#60] [Q&A] [answered]
How to translate?

**Comment by sa-si-dev** [comment, marked as answer]
You can do translation by initiating the Virtual Select with all text-related properties (Refer [Text Properties](https://sa-si-dev.github.io/virtual-select/#/properties?id=text-properties) for available properties)

To avoid code duplicates, create a project-level util function to initiate the dropdown, and define all  [Text Properties](https://sa-si-dev.github.io/virtual-select/#/properties?id=text-properties) there.

```js
function initDropdown(options) {
  return VirtualSelect.init(Object.assign({
    placeholder: 'Select',
    noOptionsText: 'No options found',

    /** define all text related properties and project-level common properties here */

  }, options));
}

initDropdown({
  ele: '#sample-select',
  options: [
    { label: 'Options 1', value: '1' },
    { label: 'Options 2', value: '2' },
    { label: 'Options 3', value: '3' },
  ],
});
```
